lib/browser/platform/windows.rb

```diff
@@ -2,8 +2,24 @@
 module Browser
   class Platform
     class Windows < Base
+      VERSION_MAP = {
+        "10.0" => "10",
+        "6.3"  => "8.1",
+        "6.2"  => "8",
+        "6.1"  => "7",
+        "6.0"  => "Vista",
+        "5.2"  => "XP x64 Edition",
+        "5.1"  => "XP",
+        "5.01" => "2000, Service Pack 1 (SP1)",
+        "5.0"  => "2000",
+        "4.0"  => "NT 4.0",
+        "98"   => "98",
+        "95"   => "95",
+      }.freeze
+
       def version
-        "0"
+        nt_version = ua[/Windows(?:\sNT)?\s*([0-9_\.]+)?/, 1]
+        VERSION_MAP[nt_version] || "0"
```

Comment:
In that case, would you use the key version extracted e.g. 6.3 for Windows 8.1 or replace that with 8.1? And what about the other version that aren't really numeric e.g. Vista/XP/2000?